outputs.tf

```diff
@@ -14,38 +14,17 @@
  * limitations under the License.
  */
 
-output "dataset_id" {
-  value       = google_bigquery_dataset.main.id
-  description = "Unique id for the dataset being provisioned"
+output "bigquery_dataset" {
+  value       = google_bigquery_dataset.main
+  description = "Bigquery dataset resource."
 }
 
-output "dataset_name" {
-  value       = google_bigquery_dataset.main.friendly_name
-  description = "Friendly name for the dataset being provisioned"
+output "bigquery_tables" {
+  value       = { for table in var.tables : table["table_id"] => google_bigquery_table.main[table["table_id"]] }
```

Comment:
Is this necessary? If we just output `google_bigquery_table.main`, can the calling configuration not access it as a map using the table IDs?

Reply:
Fair enough, updated